pom.xml

```diff
@@ -115,14 +114,10 @@
             <artifactId>lombok</artifactId>
             <version>1.18.26</version>
         </dependency>
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-        </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>bouncycastle-api</artifactId>
-            <version>2.20</version>
+            <version>2.27</version>
```

Comment:
should be in the bom and can be removed?

```suggestion
```

Reply:
yeah, version should be pulled by BOM if possible